keras_cv/ops/point_cloud.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+
+def get_rank(tensor):
+    return tensor.shape.ndims or tf.rank(tensor)
+
+
+def _center_xyzWHD_to_corner_xyz(boxes):
+    # relative corners w.r.t to origin point
```

Comment:
Please add docstrings with Args and Returns:

e.g.

      boxes: [..., num_boxes, 7] float32 Tensor for 3d boxes in [x, y, z, dx,
        dy, dz, phi].